src/cryptonote_basic/cryptonote_format_utils.cpp

```diff
@@ -1517,7 +1579,7 @@ namespace cryptonote
   bool get_block_longhash(const block& b, crypto::hash& res, uint64_t height)
   {
     const blobdata bd                 = get_block_hashing_blob(b);
-    const int hf_version              = b.major_version;
+    const uint8_t hf_version              = b.major_version;
```

Comment:
(:nauseated_face: again, but this time because there is extra padding for aligning that doesn't align).

Reply:
Fixed